### PR TITLE
Add l messages

### PR DESCRIPTION
**Pull request by xmacan**

### CODE REVIEW

**Review by netniV — CHANGES_REQUESTED** [review]

**Comment by netniV** [comment]
Still waiting on an update @xmacan ?

File: lib/utility.php [hunk 1]
```diff
@@ -101,6 +101,8 @@ function repopulate_poller_cache() {
 		ON dtr.data_input_field_id = dif.id
 		WHERE dtr.local_data_id = 0
 		GROUP BY dtr.data_template_id, dif.data_name');
+	
+	cacti_log('NOTE: Poller Cache repopulated by user ' . get_username($_SESSION['sess_user_id']), false, 'WEBUI');
```
[the remark this answers is not in the document]
Comment (xmacan) [reply lib/utility.php:105]: I overlooked your request. I'll fix it asap

[the remark this answers is not in the document]
Comment (xmacan) [reply lib/utility.php]: Fixed .... but function repopulate_poller_cache is not executed from CLI ..

File: utilities.php [hunk 2]
```diff
@@ -2124,6 +2124,8 @@ function purge_data_source_statistics() {
 	foreach($tables as $table) {
 		db_execute('TRUNCATE TABLE ' . $table);
 	}
+
+	cacti_log('NOTE: Cacti DS Stats purget by user ' . get_username($_SESSION['sess_user_id']), false, 'WEBUI');
```
[the remark this answers is not in the document]
Comment (xmacan) [reply utilities.php]: see above :-)

**Comment by TheWitness** [comment]
Actually the `install_cacti.php` may be calling it.

**Review by TheWitness — APPROVED** [review]

**Review by netniV — APPROVED** [review]

**Pull request merged**